html-forms/index.html

```diff
@@ -0,0 +1,52 @@
+<!DOCTYPE html>
+<html>
+<head>
+  <title>HTML Forms</title>
+  <style>
+    body {
+      max-width: 450px;
+      margin: auto;
+    }
+  </style>
+</head>
+<body>
+  <h1 id="title" class="text-center">Coding Form!</h1>
+  <form>
+    <label for="email">Email</label><br />
+    <input id="email" type="text" name="email"/>
+  </form>
+  <form>
```

Comment:
All of these form control elements should be within one `form` element, you should not have a separate `form` element for each section.

Reply:
Should my form elements inside of the one form element be <div> elements then to create code that is easier to read?